### PR TITLE
update instruction for branch creation lab

### LINKED ISSUE #94: FirstName-Lastname Branch lab has misleading phrase.
In order to eliminate any possible confusion, the lab where a branch must be created in Section 2, slide 4/16, the phrase "Create a branch in the repo named firstname-lastname", should be changed to "Create a branch in your repo and name it firstname-lastname using your first and last name"

### PR DESCRIPTION
**Pull request by kkemple**
Fixes #94

### CODE REVIEW

File: modules/CONT-02_Using-branches.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ screens:
       presenter-script:
         - Let's practice creating a branch.
       steps:
-        - description: "Create a branch in the repo named firstname-lastname (using your first and last names)."
+        - description: "Create a branch in your repo and name it firstname-lastname using your first and last name."
```
Comment (crichID) [inline modules/CONT-02_Using-branches.yml]: > Create a branch in **your** repo

My only issue with this change is that it is solely focused on self-serve. In the instructor led classes they will not have individual repos and they should create their branch in the class repo. This was the point of adding the class repo link to help them get to the right place. Perhaps we could say something like: "Create a branch in the class repo (linked above)"

Comment (kkemple) [inline modules/CONT-02_Using-branches.yml]: Yeah that makes sense, can we adjust that to "event repo (linked above)"?

**Comment by crichID** [comment]
Teamwork! I went ahead and changed this one too and will :shipit:

**Comment by kkemple** [comment]
:+1:

**Pull request merged**